src/test-run/commands/index.js

```diff
@@ -622,7 +623,8 @@ export function createCommandFromObject (obj) {
 }
 
 export function isCommandRejectableByPageError (command) {
-    return !isObservationCommand(command) && !isWindowManipulationCommand(command) && !isServiceCommand(command);
+    return !isObservationCommand(command) && !isWindowManipulationCommand(command) && !isServiceCommand(command) &&
+           !(command.type === TYPE.prepareBrowserManipulation && command.manipulationType === TYPE.takeScreenshot);
```

Comment:
I find it hard to parse this. Can we refactor it to make it clear when command rejectable and when not? BTW, in some places we have `browser manipulation` and in other `window manipulation`, can we come up with consistent terminology?